internal/stanza/converter.go

```diff
@@ -15,49 +15,294 @@
 package stanza
 
 import (
+	"context"
+	"encoding/json"
 	"fmt"
+	"sync"
+	"time"
 
 	"github.com/open-telemetry/opentelemetry-log-collection/entry"
 	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.uber.org/zap"
 )
 
-// Convert a stanza-style entry to a pdata.Logs
-func Convert(obsLog *entry.Entry) pdata.Logs {
+const (
+	DefaultFlushInterval = time.Second
```

Comment:
Can we use shorter interval, e.g. 100ms? Delays from different batchers add up and we end up with significant latency if we are not careful. I think 100ms is enough for high rate of collection (and probably DefaultMaxFlushCount will apply first) and for low rate of collection we don't care.